src/core/Akka/Util/FastLazy.cs

```diff
@@ -81,5 +81,80 @@ public T Value
             }
         }
     }
-}
 
+
+    /// <summary>
+    /// A fast, atomic lazy that only allows a single publish operation to happen,
+    /// but allows executions to occur concurrently.
+    /// 
+    /// Does not cache exceptions. Designed for use with <typeparamref name="T"/> types that are <see cref="IDisposable"/>
+    /// or are otherwise considered to be expensive to allocate. 
+    /// 
+    /// Read the full explanation here: https://github.com/Aaronontheweb/FastAtomicLazy#rationale
+    /// </summary>
+    /// <typeparam name="S">State type</typeparam>
+    /// <typeparam name="T">Value type</typeparam>
+    public sealed class FastLazy<S, T>
```

Comment:
Also, might want to consider contributing this change back to the separate repo I setup for the original `FastLazy` implementation in case others have this issue: https://github.com/Aaronontheweb/FastAtomicLazy/